### PR TITLE
bump platform images

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Description

> Describe the purpose of this pull request.

## PR Title

> Please make the PR title informative to the user experience. For example when adding a feature, say "Add metrics tab for Deployments" instead of "Bump Astro-UI to version X.Y.Z". Bugs are sometimes harder to make customer-facing because they are very detailed, but do what you can. Instead of "Fix metrics cardinality bug" (user does not know what 'cardinality' means and does not matter to them), say "Fix bug slowing down the metrics tab". The PR title will be used for the commit message when it's merged, and these commit messages are used to generate customer-facing release notes.

> If there is only one commit in the PR, when clicking the merge button, be sure to copy the PR title into the PR squash and merge option's commit message. By default with a single commit, it will use the commit message instead of the PR title.

## 🎟 Issue(s)

Resolves astronomer/issues#XXXX

## 🧪  Testing

> What are the main risks associated with this change, and how are they addressed by tests added in this PR?

> Please add helm unit testing, if applicable. The docs are regretfully sparse for helm unit testing, [here](https://github.com/astronomer/helm-unittest/blob/main/DOCUMENT.md) is what we have to work with. In general, these kind of tests can be used for specific assertions like 'when these values are set, the resource ends up looking like XYZ' but not for generalized assertions like 'for all resources, make sure the namespace is not set to the release name'.

> Please also add to the system testing [here](../bin/functional-tests), if applicable. This kind of testing allows you to connect into any live, running pod to make assertions about how they are operating. For example, one test connects to a Prometheus pod, queries the Prometheus API, and makes assertions about the Prometheus scrape targets being healthy.

## 📸 Screenshots

> Add screenshots to illustrate the changes, if applicable.

## 📋 Checklist

- [ ] The PR title is informative to the user experience

### CODE REVIEW

File: charts/astronomer/values.yaml [hunk 1]
```diff
@@ -15,19 +15,19 @@ tolerations: []
...
     pullPolicy: IfNotPresent
   registry:
     repository: astronomerinc/ap-registry
     tag: 3.11.5-4
     pullPolicy: IfNotPresent
   houston:
-    repository: astronomerinc/ap-houston-api
-    tag: 0.20.9
+    repository: asoldatenko/ap-houston-api
+    tag: 0.21.3
```
Comment (andriisoldatenko) [inline charts/astronomer/values.yaml:26]: need wait for houston-api image:
https://app.circleci.com/pipelines/github/astronomer/houston-api/2170/workflows/270de444-9c90-4138-85b8-316eb283e19a/jobs/7300

File: charts/astronomer/values.yaml [hunk 2]
```diff
@@ -15,19 +15,19 @@ tolerations: []
...
     pullPolicy: IfNotPresent
   astroUI:
     repository: astronomerinc/ap-astro-ui
-    tag: 0.20.2
+    tag: 0.21.0
```
Comment (andriisoldatenko) [inline charts/astronomer/values.yaml:30]: based on https://app.circleci.com/pipelines/github/astronomer/astro-ui/1878/workflows/6bcad779-eb5a-4ed3-9d2c-7f38a369e31e/jobs/4191

File: charts/astronomer/values.yaml [hunk 3]
```diff
@@ -15,19 +15,19 @@ tolerations: []
 images:
   commander:
     repository: astronomerinc/ap-commander
-    tag: 0.18.0
+    tag: 0.21.0
```
Comment (andriisoldatenko) [inline charts/astronomer/values.yaml:18]: not ready https://app.circleci.com/pipelines/github/astronomer/commander/180/workflows/7b348c6b-ad8d-4329-b08c-07f43fa7d7f0/jobs/596

**Comment by jpweber** [comment]
We have a helm unites failing because it is looking for a specific docker image tag version. We shouldn't probably test on that. @andriisoldatenko can you just remove this file.  `charts/astronomer/tests/deployment_test.yaml` and push again please?

File: charts/astronomer/values.yaml (hunk 3 above)
Comment (andriisoldatenko) [reply charts/astronomer/values.yaml:18]: done.

**Comment by jpweber** [comment]
We have a helm unites failing because it is looking for a specific docker image tag version. We shouldn't probably test on that. @andriisoldatenko can you edit `charts/astronomer/tests/deployment_test.yaml` and remove the test

**Comment by andriisoldatenko** [comment]
> charts/astronomer/tests/deployment_test.yaml

i can fix.

File: charts/astronomer/values.yaml (hunk 2 above)
Comment (adam2k) [reply charts/astronomer/values.yaml:30]: I think @samblackk is creating a new release for this, so it will be `0.21.1`.

**Review by jpweber — APPROVED** [review]
Resolve conflicts and this looks good to go.

File: charts/astronomer/values.yaml (hunk 2 above)
Comment (andriisoldatenko) [reply charts/astronomer/values.yaml]: done

**Pull request merged**